src/wrap.js

```diff
@@ -29,7 +29,7 @@ var arity = require('./arity');
  *
```

Comment:
@anton-rudeshko: I guess I'm willing to, if you feel strongly about it.  I don't see much of a need.  Part of it is that this function is not one I use often, so I'm not particularly invested in it.  Another part is that I'm (still!) generally less interested in inline documentation.  :-)

Do you have a suggested short example?